apps/parsley/src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -27,14 +33,30 @@ export const EvergreenTaskSubHeader: React.FC<Props> = ({
   testID,
 }) => {
   const { sendEvent } = usePreferencesAnalytics();
-  const { loading, task } = useTaskQuery({
+  const { loading: isLoadingTask, task: taskData } = useTaskQuery({
     buildID,
     execution,
     logType,
     taskID,
   });
 
-  if (loading || !task) {
+  const { data: testData, loading: isLoadingTest } = useQuery<
+    TestLogUrlAndRenderingTypeQuery,
+    TestLogUrlAndRenderingTypeQueryVariables
+  >(GET_TEST_LOG_URL_AND_RENDERING_TYPE, {
+    skip: !(
+      logType === LogTypes.EVERGREEN_TEST_LOGS &&
```

Comment:
I think this might be the only thing we really need to specify everything else should be defined afaict.